whisper/mailserver/server_test.go

```diff
@@ -167,7 +167,8 @@ func singleRequest(t *testing.T, server *WMailServer, env *whisper.Envelope, p *
 
 	src[0]++
 	ok, lower, upper, topic = server.validateRequest(src, request)
-	if ok {
+	if !ok {
+		// request should be valid regardless of signature
 		t.Fatalf("request validation false positive, seed: %d (lower: %d, upper: %d).", seed, lower, upper)
```

Comment:
then it's no longer a false positive, but you should also somehow check that indeed it failed because of the signature.

Reply:
fixed the error info